app/views/logs/_tasklist.html.erb

```diff
@@ -1,5 +1,6 @@
 <ol class="app-task-list govuk-!-margin-top-8">
-  <% @log.form.sections.map do |section| %>
+  <% @log.form.sections.each do |section| %>
+    <% next if section.subsections.none? { |subsection| subsection.displayed_in_tasklist?(@log) } %>
```

Comment:
Can this be extracted into a section model method?

Reply:
Yes, updated